test/e2e/connectivity_test.go

```diff
@@ -152,7 +152,7 @@ func testPodConnectivityOnSameNode(t *testing.T, data *TestData) {
 func (data *TestData) testHostPortPodConnectivity(t *testing.T, clientNamespace, serverNamespace string) {
 	// Create a server Pod with hostPort set to 80.
 	hpPodName := randName("test-host-port-pod-")
-	hpPodPort := int32(80)
+	hpPodPort := int32(140)
```

Comment:
Please add comments to explain why we need change port number here, and this change should only be applied in rancher environment.